install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,118 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
```

Comment:
Stating "latest release" could be a challenge to update along with the docs. Perhaps something like, "As of release x.x...." would be better.

Reply:
s/has updated/updated